tests/models/plants/test_plants_model.py

```diff
@@ -91,14 +91,13 @@ def test_PlantsModel_update_canopy_layers(
     # layer_absorbed_irradiation, which should not have been regenerated yet
     del fixture_canopy_layer_data["layer_absorbed_irradiation"]
     for layer_name, layer_vals, layer_indices in fixture_canopy_layer_data.values():
-        expected = fixture_core_components.layer_structure.from_template()
+        expected = from_template()
         expected[layer_indices] = layer_vals[:, None]
         xarray.testing.assert_allclose(fxt_plants_model.data[layer_name], expected)
 
     # Check layer_absorbed_irradiation is indeed still empty
     xarray.testing.assert_allclose(
-        fxt_plants_model.data["layer_absorbed_irradiation"],
-        fixture_core_components.layer_structure.from_template(),
+        fxt_plants_model.data["layer_absorbed_irradiation"], from_template()
```

Comment:
Did we not rename this variable in the context of the variables PR?

Reply:
We did but they're parallel branches! I'm going to merge and then update the `variables_1` branch.